conflux/adapter.js

```diff
@@ -0,0 +1,88 @@
+const Web3Utils = require('web3-utils')
+const { Conflux } = require('js-conflux-sdk')
+const { Requester, Validator } = require('@chainlink/external-adapter')
+require('dotenv').config()
+
+const provider = new Conflux({
+  url: process.env.URL,
+  logger: console, //JSON RPC call logging
+  networkId: Number(process.env.NETWORK_ID),
+  defaultGasRatio: 1.3,
+  defaultStorageRatio: 1.3
+})
+const privateKey = process.env.PRIVATE_KEY
```

Comment:
Almost every configuration variables should come from `config.ts`, and received as parameters from the execute functions. Use the `example` adapter as reference.